maintainers/maintainer-list.nix

```diff
@@ -8344,6 +8344,13 @@
     githubId = 2422454;
     name = "Kai Wohlfahrt";
   };
+  kylehendricks = {
+    name = "Kyle Hendricks";
+    email = "kyle-github@mail.hendricks.nu";
+
```

Comment:
```suggestion
```

Reply:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.